rascal-vscode-extension/src/RascalExtension.ts

```diff
@@ -182,5 +196,17 @@ function calculateRascalREPLMemory() {
     if (os.totalmem() >= gb(8)) {
         return "-Xmx2400M";
     }
-    return "-Xmx800M";
+    return "-Xmx800M";    
+}
+
+function calculateRascalREPLStackSize() {
+    const config = vscode.workspace.getConfiguration();
+    if (config.has('rascal.interpreter.stackSize')) {
+        const stackSize = config.get('rascal.interpreter.stackSize');
+        if (stackSize !== null) {
+            return `-Xss${stackSize}M`;
+        }
+    }
+
+    return "";    
```

Comment:
return the default here?

Reply:
This goes with the comment above -- if we return `""`, the code that starts the shell checks and does not include the `-Xss` flag. This then falls back to the default used by the JVM.